### PR TITLE
Added logout functionality

### PR DESCRIPTION
**Pull request by gcoomber**

### CODE REVIEW

File: TextCrunch/MailboxViewController.swift [hunk 1]
```diff
@@ -122,6 +140,10 @@ class MailboxViewController: UIViewController, 	UITableViewDataSource, UITableVi
 			svc.listing = selectedNegotiation.listing
 			svc.isNewListing = false
 			svc.seguedFromMailbox = true
+		} else if (segue.identifier == "LogoutSegue") {
+			var test = 1
+			test = 2
+			test = 3 * test
```
Comment (hpabst) [inline TextCrunch/MailboxViewController.swift]: This stuff should be deleted before merging.

File: TextCrunch/NegotiationViewController.swift [hunk 2]
```diff
@@ -135,43 +136,44 @@ class NegotiationViewController : UIViewController, UITableViewDataSource, UITab
...
 
-            let push = PFPush()
-            push.setQuery(pushQuery)
-            push.setMessage(message.content)
-            push.sendPushInBackground()*/
-		}
+				let push = PFPush()
+				push.setQuery(pushQuery)
+				push.setMessage(message.content)
+				push.sendPushInBackground()*/
```
Comment (hpabst) [inline TextCrunch/NegotiationViewController.swift]: The commented out code should be deleted unless you need it here for future development.

Comment (gcoomber) [inline TextCrunch/NegotiationViewController.swift]: I think this is derek's. Darek do you want this here?

File: TextCrunch/MailboxViewController.swift (hunk 1 above)
Comment (gcoomber) [inline TextCrunch/MailboxViewController.swift]: Good catch. Done

**Pull request merged**